src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureDrawing,AreaCalculator {

```

Comment:
```suggestion
public abstract class Figure implements FigureDrawing, AreaCalculator {
```
make sure you use auto-formatter before committing